datarepo-clienttests/build.gradle

```diff
@@ -38,7 +39,7 @@ dependencies {
         swaggerAnnotations = "2.1.5"
         jersey = "2.30.1"
 
-        datarepoClient = "1.46.0-SNAPSHOT"
+        datarepoClient = "1.53.0-LOCAL"
```

Comment:
Ah shoot, that snuck in there and shouldn't be. This was me publishing the changes to my local maven repository just to make fix up all the tests and make sure the deprecated code works. I think we need to bump the version after we merge to develop, since we have an action to publish the client code, right?

Reply:
Ah, ok. Yes, the action will bump the version. (But I think it might not touch the '-SNAPSHOT' part, so I think we need to switch back from LOCAL before merging)

Reply:
I had put `-LOCAL` on there just to distinguish that it was referring to a deployment that only existed on my laptop. I've reverted it back to `"1.46.0-SNAPSHOT"` and we'll let the action deal with the bumping of the version